docs/Installation/Re-Installing_Zenko.rst

```diff
@@ -0,0 +1,113 @@
+  2018-07-04 01:01:11,359 [myid:2] - INFO  [WorkerReceiver[myid=2]:FastLeaderElection@600] - Notification: 1 (message format version), 2 (n.leader), 0x10000009e (n.zxid), 0x584 (n.round), LOOKING (n.state), 2 (n.sid), 0x1 (n.peerEpoch) LOOKING (my state)
+  2018-07-04 01:01:11,360 [myid:2] - WARN  [WorkerSender[myid=2]:QuorumCnxManager@588] - Cannot open channel to 4 at election address zenko-zenko-quorum-3.zenko-zenko-quorum-headless.default.svc.cluster.local/10.233.99.143:3888
+  java.net.ConnectException: Connection refused (Connection refused)
+        at java.net.PlainSocketImpl.socketConnect(Native Method)
+        at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
+        at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
+        at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
+        at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
+        at java.net.Socket.connect(Socket.java:589)
+        at org.apache.zookeeper.server.quorum.QuorumCnxManager.connectOne(QuorumCnxManager.java:562)
+        at org.apache.zookeeper.server.quorum.QuorumCnxManager.toSend(QuorumCnxManager.java:538)
+        at org.apache.zookeeper.server.quorum.FastLeaderElection$Messenger$WorkerSender.process(FastLeaderElection.java:452)
+        at org.apache.zookeeper.server.quorum.FastLeaderElection$Messenger$WorkerSender.run(FastLeaderElection.java:433)
+        at java.lang.Thread.run(Thread.java:748)
+  2018-07-04 01:01:11,360 [myid:2] - INFO  [WorkerSender[myid=2]:QuorumPeer$QuorumServer@167] - Resolved hostname: zenko-zenko-quorum-3.zenko-zenko-quorum-headless.default.svc.cluster.local to address: zenko-zenko-quorum-3.zenko-zenko-quorum-headless.default.svc.cluster.local/10.233.99.143
+  2018-07-04 01:01:11,360 [myid:2] - INFO  [WorkerSender[myid=2]:QuorumCnxManager@337] - Have smaller server identifier, so dropping the connection: (5, 2)
+  2018-07-04 01:01:11,364 [myid:2] - INFO  [WorkerReceiver[myid=2]:FastLeaderElection@600] - Notification: 1 (message format version), 2 (n.leader), 0x10000009e (n.zxid), 0x584 (n.round), LOOKING (n.state), 1 (n.sid), 0x1 (n.peerEpoch) LOOKING (my state)
+  2018-07-04 01:01:12,108 [myid:2] - INFO  [NIOServerCxn.Factory:0.0.0.0/0.0.0.0:2181:NIOServerCnxnFactory@192] - Accepted socket connection from /10.233.114.70:45648
+  2018-07-04 01:01:12,108 [myid:2] - WARN  [NIOServerCxn.Factory:0.0.0.0/0.0.0.0:2181:NIOServerCnxn@373] - Exception causing close of session 0x0 due to java.io.IOException: ZooKeeperServer not running
+  2018-07-04 01:01:12,109 [myid:2] - INFO  [NIOServerCxn.Factory:0.0.0.0/0.0.0.0:2181:NIOServerCnxn@1044] - Closed socket connection for client /10.233.114.70:45648 (no session established for client)
+  2018-07-04 01:01:12,712 [myid:2] - INFO  [NIOServerCxn.Factory:0.0.0.0/0.0.0.0:2181:NIOServerCnxnFactory@192] - Accepted socket connection from /10.233.114.70:45656
+
+Purge Old Objects
```

Comment:
A lot of these is what you want to clear out in the case of cleaning up a MetalK8s install after uninstalling Zenko but not really specific to Zenko.

Reply:
I'm in a bit of a jam here. We want to offer Zenko and MetalK8s together(-ish), but to warn people when they play poorly with each other, I need to put the information into either a Zenko bucket or a MetalK8s bucket. This looks like a case of Zenko and MetalK8s not getting along together. Given this dilemma, I think Zenko is the best place to drop the information. What I can tell from 655 and 1057, and from what you've told me, is that reinstalling Zenko requires us to destroy volumes in a MetalK8s cluster (ZENKO-655) and to delete a mess of files (ZENKO-1057), that appear to my untutored eye to be Zenko-specific. Is this true for any software that would use MetalK8s?

Reply:
I'm not arguing the validity of having MetalK8s info here. I'm all for keeping a single doc that has Zenko on MetalK8s specific instructions but these are instructions for cleaning up a MetalK8s install and would really be better under something called "Restoring servers to original state before re installing MetalK8s".

Reply:
When re-installing Zenko, why would you be purging your Kubernetes cluster at all?!